client/content_publishing_report/controllers/ContentPublishingReportController.js

```diff
@@ -342,7 +344,14 @@ export const generateTitle = (chart, params) => {
         return params.chart.title;
     }
 
-    const parentField = _.get(params, 'aggs.group.field');
+    let parentField = _.get(params, 'aggs.group.field');
+
+    if (parentField.startsWith('{"scheme')) {
```

Comment:
So translations work, you might need to use `superdeskApi.entities.vocabulary.getCustomFieldVocabularies()`, and use the current users language to get the translated name for the vocabulary